x-pack/plugins/security_solution/public/resolver/view/index.tsx

```diff
@@ -123,17 +126,28 @@ export const Resolver = React.memo(function Resolver({
           tabIndex={0}
           aria-activedescendant={activeDescendantId || undefined}
         >
-          {edgeLineSegments.map(({ points: [startPosition, endPosition], metadata }, index) => (
-            <EdgeLine
-              edgeLineMetadata={metadata}
-              key={index}
-              startPosition={startPosition}
-              endPosition={endPosition}
-              projectionMatrix={projectionMatrix}
-            />
-          ))}
-          {[...processNodePositions].map(([processEvent, position], index) => {
-            const adjacentNodeMap = processToAdjacencyMap.get(processEvent);
+          {connectingEdgeLineSegments.map(
+            (
+              {
+                entity: {
+                  points: [startPosition, endPosition],
+                  metadata,
+                },
+              },
+              index
+            ) => (
+              <EdgeLine
+                edgeLineMetadata={metadata}
+                key={index}
```

Comment:
Because `rbush` might return these entities in a non-stable order, we should use a stable `key`. This will help react reconcile old and new components. Since this runs every frame.... :)